### PR TITLE
0.2.0 changelog and backward compatibility explainers

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,74 +7,160 @@ v0.2.0 - 2020-xx-xx
...
+    Any instances of lifting nodes by tag property without the tag will need to be updated to include the tag name.
 
-- user.admin -> user.isAdmin()
+Removed Insecure Mode
+~~~~~~~~~~~~~~~~~~~~~
+
+What changed
+    The "insecure" option in cell.yaml has been removed.
+
+Why make the change
```
Comment (vEpiphyte) [inline CHANGELOG.rst]: It is also possible to bootstrap a root password directly via command line / envar / cell.yaml

File: CHANGELOG.rst [hunk 2]
```diff
@@ -7,74 +7,160 @@ v0.2.0 - 2020-xx-xx
...
+
+What you need to do
+    If you have custom model elements that have default values, they will no longer be populated by default.  As a work around, you may create a trigger which populates the property when the node is added, but use caution when merging properties from multiple layers when populating defaults.
+
+Additional Changes
+------------------
+
+- map_async is now enabled by default for all slabs
+- Synapse tools may not be used across minor versions.
```
Comment (vEpiphyte) [inline CHANGELOG.rst]: we should enumerate this a bit more.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -7,74 +7,160 @@ v0.2.0 - 2020-xx-xx
...
-- Remote layer removed
+What changed
+    The various splice events provided by the storm runtime have been consolidated and streamlined into a single ``node:edits`` event type.  The new ``node:edits`` events contain a more compressed and aggregated representation of node changes and include changes that were previous unreported.  Callers may now specify the format for the node edits reported by the storm runtime by using the ``editformat`` optional parameter to the storm runtime, but the default output will now be ``node:edits`` rather than splices.  The following table enumerates the options for the ``editformat`` parameter:
 
-- Deprecation of lots of methods
+    ========== =========================================================
+    editformat Storm runtime behavior
+    ========== =========================================================
+    nodeedits  Provide new ``node:edits`` events.
+    splices    Provide backward compatible splice events.
```
Comment (vEpiphyte) [inline CHANGELOG.rst]: provide `0.1.x` backward compatible splice events.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -7,74 +7,160 @@ v0.2.0 - 2020-xx-xx
...
+    count      Provide ``node:edits:count`` events with simple counts.
+    none       Do not transmit any representation for node edits.
+    ========== =========================================================
 
-- Removed cortex offset storage
+Why make the change
+    The splice format was originally designed to facilitate a single atomic edit per-splice.  As such, it required the potentially large primary property value to be embedded in each splice.  When making multiple edits, this representation is inefficient and causes the retransmission, and potential storage, of duplicate data.  Additionally, the key-value structure of the splice format provided unnecessary extensibility at the cost of transmission/storage size.
+
+What you need to do
+    Update any code that consumes/indexes the various splice events to handle the new ``node:edits`` format.  Additionally, callers may specify ``editformat: "splices"`` within their storm runtime options to enable backward compatible splice generation.
```
Comment (jnwatson) [inline CHANGELOG.rst]: backward-compatible

File: CHANGELOG.rst (hunk 2 above)
Comment (jnwatson) [reply CHANGELOG.rst]: (Our minor version is really a major version)

File: CHANGELOG.rst (hunk 4 above)
Comment (invisig0th) [reply CHANGELOG.rst]: Not according to merriam-webster and most of the internet...

File: CHANGELOG.rst (hunk 2 above)
Comment (invisig0th) [reply CHANGELOG.rst]: agreed, i'll make it specific to using tools to connect to services of a different minor version.

File: CHANGELOG.rst (hunk 4 above)
Comment (invisig0th) [reply CHANGELOG.rst]: done

**Pull request merged**